CTestJenkins.cmake

```diff
@@ -73,13 +74,13 @@ set(CTEST_CONFIGURATION_TYPE ${CTEST_BUILD_CONFIGURATION})
 ctest_start("${JENKINS_DASHBOARD}" ${QUIET_})
 ctest_configure(RETURN_VALUE CONFIGURING_RET_VALUE ${QUIET_})
 ctest_build(RETURN_VALUE BUILDING_RET_VALUE ${QUIET_})
-ctest_test(${QUIET_})
-if(CTEST_COVERAGE_COMMAND AND NOT DISABLE_CTEST_COVERAGE)
-    ctest_coverage(${QUIET_})
-endif()
-if(CTEST_MEMORYCHECK_COMMAND AND NOT DISABLE_CTEST_MEMORYCHECK)
-    ctest_memcheck(EXCLUDE_LABEL NoMemoryCheck ${QUIET_})
-endif()
+# ctest_test(${QUIET_})
```

Comment:
This comment was for testing. Remove it.

Reply:
Removed